R/Utils.R

```diff
@@ -262,6 +277,11 @@ is_parameterised <- function(ps) {
   rJava::.jcall(md, "I", "getParameterCount") > 0
 }
 
+num_parameters <- function(ps) {
```

Comment:
Sorry let me try to be clearer: you can refactor `is_parameterised()` to call `num_parameters()`:

```R
is_parameterised <- function(ps) {
  num_parameters(ps) > 0L
}
```